core/src/main/scala/sttp/client4/internal/ToRfc2616Converter.scala

```diff
@@ -12,15 +12,25 @@ private[client4] object ToRfc2616Converter {
   def requestToRfc2616(request: GenericRequest[_, _], sensitiveHeaders: Set[String]): String =
     apply(request, sensitiveHeaders)
 
+  def requestToRfc2616(
```

Comment:
same here, it's an `internal` class, we can just have a single method with all params